Assets/Scripts/SpawnObjects.cs

```diff
@@ -0,0 +1,43 @@
+// using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpawnObjects : MonoBehaviour
+{
+    public GameObject[] objects;
+
+    public float spawnInterval = 2.0f;
+    public float minSpawnInterval = 0.3f;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        StartCoroutine(Spawn());
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+    }
+    IEnumerator Spawn()
+    {
+        while (true)
+        {
+            int spawnRandom = Random.Range(0, 1000) % objects.Length;
```

Comment:
```suggestion
            int spawnRandom = Random.Range(0, objects.Length);
```
이렇게 하시는건 어떨지!?
아래쪽도 마찬가지요